src/split-panel/bottom.tsx

```diff
@@ -59,23 +53,29 @@ export function SplitPanelContentBottom({
   return (
     <div
       {...baseProps}
-      className={clsx(baseProps.className, styles.drawer, styles['position-bottom'], testUtilStyles.root, {
-        [testUtilStyles['open-position-bottom']]: isOpen,
-        [styles['drawer-closed']]: !isOpen,
-        [styles['drawer-mobile']]: isMobile,
-        [styles['drawer-disable-content-paddings']]: disableContentPaddings,
-        [styles.animating]: isRefresh && (state === 'entering' || state === 'exiting'),
```

Comment:
This change impacts the non-toolbar version as well. Is it expected?

Reply:
Refresh (non-toolbar) version now gets animations via the `.with-motion` class too. I checked manually that the split panel still has motion upon entering with and without the toolbar in VR. You can verify on [this dev page](https://d21d5uik3ws71m.cloudfront.net/components/bed78a5d67a934450dbdc2ed470148103d9bdd22/dev-pages/index.html#/light/app-layout/with-split-panel). Is there somewhere you saw the animations not being applied as expected?

Reply:
If you tested this in non-toolbar-VR, it is all good. Just needed to call out the impact radius of this PR